src/app/server/Dnssd.cpp

```diff
@@ -213,6 +221,9 @@ CHIP_ERROR DnssdServer::AdvertiseOperational()
         AddICDKeyToAdvertisement(advertiseParameters);
 #endif
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
```

Comment:
Is it in facttrue that if INET_CONFIG_ENABLE_TCP_ENDPOINT is set we are in fact listening for TCP connections?  It used to very much not be true, but maybe that's changed now?